mono/mini/m2n-gen.rb

```diff
@@ -0,0 +1,137 @@
+cookies = [
```

Comment:
Isn't it something the C preprocessor would be very good at?

Reply:
This would be horribly complicated to do it with C preprocessor.

Reply:
I think we can use newer stuff in the sdk code, don't have to stick with shell+make.

Reply:
The new codegen script could use better docs and make target, though.

Reply:
@vargaz this wouldn't only be used by the sdks code, as it would be used for any other project that would want to compile mono for webassembly

@kumpera  couldn't we use C# for that script then? If this script is only used for webassembly, I think it's fair enough to consider that we are not going to bootstrap mono, and so a system mono would be available and usable.